docs/GettingStarted.md

```diff
@@ -548,17 +515,32 @@ require 'active_job'
 require 'ddtrace'
 
 Datadog.configure do |c|
-  c.use :active_job, options
+  c.use :active_job
 end
 
 ExampleJob.perform_later
 ```
 
-Where `options` is an optional `Hash` that accepts the following parameters:
```

Comment:
Is the `log_injection` parameter no long relevant for Active Job?

Reply:
Correct; it has been deprecated. That's my understanding why it wasn't in the docs here.